paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -476,6 +476,60 @@ class PaparazziPluginTest {
     snapshotsDir.deleteRecursively()
   }
 
+  @Test
+  fun rerunTestsOnRecordPropertyChange() {
+    val fixtureRoot = File("src/test/projects/rerun-after-test-record")
+
+    // Take 1
+    val firstRunResult = gradleRunner
+      .withArguments("testDebugUnitTest", "--stacktrace")
+      .forwardOutput()
+      .runFixture(fixtureRoot) { build() }
+
+    with(firstRunResult.task(":testDebugUnitTest")) {
+      assertThat(this).isNotNull()
+      assertThat(this!!.outcome).isEqualTo(SUCCESS)
+    }
+
+    // Take 2
+    val secondRunResult = gradleRunner
+      .withArguments("recordPaparazziDebug", "--stacktrace")
+      .forwardOutput()
+      .runFixture(fixtureRoot) { build() }
+
+    with(secondRunResult.task(":testDebugUnitTest")) {
+      assertThat(this).isNotNull()
+      assertThat(this!!.outcome).isEqualTo(SUCCESS) // not UP-TO-DATE
+    }
+  }
+
+  @Test
+  fun rerunTestsOnVerifyPropertyChange() {
```

Comment:
can we group this into one test `rerunTestsOnPropertyChange` with one project `rerun-property-change` merging the 3 takes, i.e., testDebugUnitTest, recordPaparazziDebug, verifyPaparazziDebug?